src/turbomind/models/llama/LlamaBatch.cc

```diff
@@ -932,6 +931,7 @@ void LlamaBatch<T>::FreeBuffer()
 template<typename T>
 LlamaBatch<T>::LlamaBatch(const EngineParams& params, int cache_block_seq_len, int quant_policy, LlamaV2<T>* model):
     max_batch_size_(params.max_batch_size),
+    max_forward_token_num_(params.max_prefill_token_num + params.max_batch_size),
```

Comment:
I didn't get  "+ params.max_batch_size", why?

Reply:
There may be `max_batch_size - 1` decoding sequences and 1 prefilling sequence with `max_prefill_token_num` tokens.

Reply:
Get it